package.json

```
@@ -10,6 +10,7 @@
     "watch-node": "nodemon dist/server.js",
     "watch": "concurrently -k -p \"[{name}]\" -n \"TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-ts\" \"npm run watch-node\"",
     "test": "ts-mocha -p tsconfig.json test/**/*.spec.ts",
+    "integ-test": "ts-mocha -p tsconfig.json integration_test/**/*.spec.ts",
```

lets keep the test folder and do 
```suggestion
    "integ-test": "ts-mocha -p tsconfig.json test/integration/**/*.spec.ts",
```